examples.md

```diff
@@ -422,42 +426,32 @@ When dependencies are installed later in the workflow, we must specify the same
 ## Rust - Cargo
 
 ```yaml
-- name: Cache cargo registry
-  uses: actions/cache@v1
-  with:
-    path: ~/.cargo/registry
-    key: ${{ runner.os }}-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
-- name: Cache cargo index
-  uses: actions/cache@v1
+- name: Cache cargo
```

Comment:
We could optionally just remove this `name` from the example, it was only needed to differentiate the multiple cache steps and most examples don't specify a name.